src/lib/context-menu.ts

```diff
@@ -124,8 +124,19 @@ export class ContextMenu {
           const itemTitle = sectionTitle
             ? `“${this.trimTitle(sectionTitle, 30)}”`
             : 'This Section'
-
-          if (types.has('section')) {
+          if (
+            types.has('section') &&
+            !isChildOfNodeType(
+              this.view.state.doc,
+              $pos.pos,
+              schema.nodes.abstracts.name
+            ) &&
+            !isChildOfNodeType(
```

Comment:
looks good, additionally you can allow passing an array for parentNodeType and call this function here just once, which would make for a neater look. Also you can consider passing  nodeType instead of nodeType.name for the sake of stricter type constraints as string is quite generic. But that's optional.

Reply:
Done